src/search/parse/parse.go

```diff
@@ -21,7 +22,12 @@ func ParseURL(rawURL string) string {
 		log.Error().Err(err).Msgf("Couldn't parse URL: %v", rawURL)
 		return rawURL
 	}
-	return parsedURL.String()
+
+	urlString := parsedURL.String()
+	if len(urlString) != 0 && len(parsedURL.Path) == 0 { // https://fast.com -> https://fast.com/
```

Comment:
Should probably change the example to `https://brzaguza.rs -> https://brzaguza.rs/`